apps/web/components/eventtype/AvailabilityTab.tsx

```diff
@@ -131,67 +190,14 @@ const EventTypeSchedule = () => {
           )}
         />
       </div>
-      <div className="space-y-4 rounded border px-6 pb-4">
-        <ol className="table border-collapse text-sm">
-          {weekdayNames(i18n.language, 1, "long").map((day, index) => {
-            const isAvailable = !!filterDays(index).length;
-            return (
-              <li key={day} className="my-6 flex border-transparent last:mb-2">
-                <span
-                  className={classNames(
-                    "w-20 font-medium sm:w-32",
-                    !isAvailable && "text-gray-500 opacity-50"
-                  )}>
-                  {day}
-                </span>
-                {isLoading ? (
-                  <SkeletonText className="block h-5 w-60" />
-                ) : isAvailable ? (
-                  <div className="space-y-3 text-right">
-                    {filterDays(index).map((dayRange, i) => (
-                      <div key={i} className="flex items-center leading-4">
-                        <span className="w-16 sm:w-28 sm:text-left">
-                          {format(dayRange.startTime, timeFormat === 12)}
-                        </span>
-                        <span className="ltr:ml-4 rtl:mr-4">-</span>
-                        <div className="ml-6">{format(dayRange.endTime, timeFormat === 12)}</div>
-                      </div>
-                    ))}
-                  </div>
-                ) : (
-                  <span className="ml-6 text-gray-500 opacity-50 sm:ml-0">{t("unavailable")}</span>
-                )}
-              </li>
-            );
-          })}
-        </ol>
-        <hr />
-        <div className="flex flex-col justify-center gap-2 sm:flex-row sm:justify-between">
-          <span className="flex items-center justify-center text-sm text-gray-600 sm:justify-start">
-            <FiGlobe className="ltr:mr-2 rtl:ml-2" />
-            {schedule?.timeZone || <SkeletonText className="block h-5 w-32" />}
-          </span>
-          <Button
-            href={`/availability/${schedule?.schedule.id}`}
-            color="minimal"
-            EndIcon={FiExternalLink}
-            target="_blank"
-            rel="noopener noreferrer">
-            {t("edit_availability")}
-          </Button>
-        </div>
-      </div>
+      <EventTypeScheduleDetails />
```

Comment:
Adding this as a separate component prevents the AvailabilitySelect from being re-rendered.